pkg/module/go_get_fetcher.go

```diff
@@ -0,0 +1,163 @@
+package module
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	pkgerrors "github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+var (
+	// ErrLimitExceeded signals that github.com refused to serve the request due to exceeded quota
+	ErrLimitExceeded = errors.New("github limit exceeded")
```

Comment:
This seems to be not used anywhere, furthermore it seems that `goget.go` and goget_test.go` are still around. Can we delete them in this pr?

Reply:
It was used on ~129 after the `isLimitHit` check